src/pages/home/home.module.scss

```
@@ -0,0 +1,52 @@
+body, h1, h2, h3, h4, h5, h6, p, blockquote, pre, dl, dd, ol, ul, figure, hr {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Arial, sans-serif;
+}
+
+
+*, ::before, ::after {
+  box-sizing: border-box;
+}
+
+.Home {
+  display: grid;
+  color: #f2f2f2;
+}
+
+.NavBar {
+  display: flex;
+  flex-direction:row;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 50px;
+  background-color: rgb(117, 31, 245);
+  
+}
+
+.Logo {
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  margin-left: 10px;
+}
+
+.SectionContainer {
+  display: grid;
+  justify-self: center;
+  align-items: center;
+  grid-template-columns: 1fr;
+  gap: 10px; 
+  width: 60vw;
+  height: full;
+  margin-top: 100px;
+  background-color: rgb(138, 151, 251);
+}
+.Sections{
```

Faltou um espaço aqui.

```suggestion
}

.Sections {
```